src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html

```diff
@@ -63,3 +63,11 @@
                     [name]="showCancelButtonLabel()"></cd-back-button>
   </div>
 </div>
+
+<ng-template #skipConfirmTpl>
+  <span i18n>You are about to skip the cluster expansion process.
+             You’ll need to <strong>navigate through the menu to add hosts and services</strong></span>
+  <br>
+  <br>
```

Comment:
IMHO it's more sustainable to [set margin/padding/...](https://getbootstrap.com/docs/4.5/utilities/spacing/) than to add a chain of <br />.

Reply:
Done, thanks